WooCommerce/Classes/ViewRelated/JetpackSetup/WPComLogin/WPComEmailLoginView.swift

```diff
@@ -0,0 +1,128 @@
+    @ObservedObject private var viewModel: WPComEmailLoginViewModel
+    @FocusState private var isEmailFieldFocused: Bool
+
+    /// The closure to be triggered when the Install Jetpack button is tapped.
+    private let onSubmit: (String) -> Void
+
+    init(viewModel: WPComEmailLoginViewModel,
+         onSubmit: @escaping (String) -> Void) {
+        self.viewModel = viewModel
+        self.onSubmit = onSubmit
+    }
+
+    var body: some View {
+        ScrollView {
+            VStack(alignment: .leading, spacing: Constants.blockVerticalPadding) {
+                JetpackInstallHeaderView()
+
+                // title and description
+                VStack(alignment: .leading, spacing: Constants.contentVerticalSpacing) {
+                    Text(viewModel.titleString)
+                        .largeTitleStyle()
+                    Text(viewModel.subtitleString)
+                        .subheadlineStyle()
+                }
+
+                // Email field
+                AccountCreationFormFieldView(viewModel: .init(
+                    header: Localization.emailLabel,
+                    placeholder: Localization.enterEmail,
+                    keyboardType: .emailAddress,
+                    text: $viewModel.emailAddress,
+                    isSecure: false,
+                    errorMessage: nil,
+                    isFocused: isEmailFieldFocused
+                ))
+                .focused($isEmailFieldFocused)
+
+                Spacer()
+            }
+            .padding(Constants.contentPadding)
+        }
+        .safeAreaInset(edge: .bottom) {
+            VStack {
+                // Primary CTA
+                Button(viewModel.titleString) {
+                    onSubmit(viewModel.emailAddress)
+                }
+                .buttonStyle(PrimaryButtonStyle())
+                .disabled(!viewModel.isEmailValid)
+
+                // Terms label
+                AttributedText(viewModel.termsAttributedString)
+            }
+            .padding(Constants.contentPadding)
+            .background(Color(uiColor: .systemBackground))
+        }
+    }
+}
+
+private extension WPComEmailLoginView {
+    enum Constants {
+        static let blockVerticalPadding: CGFloat = 32
+        static let contentVerticalSpacing: CGFloat = 8
+        static let contentPadding: CGFloat = 16
+    }
+
+    enum Localization {
+        static let emailLabel = NSLocalizedString(
+            "Email address",
+            comment: "Label for the email field on the WPCom email login screen of the Jetpack setup flow."
+        )
+        static let enterEmail = NSLocalizedString(
+            "Enter email",
+            comment: "Placeholder text for the email field on the WPCom email login screen of the Jetpack setup flow."
+        )
+    }
+}
+
+
+struct WPComEmailLoginView_Previews: PreviewProvider {
+    static var previews: some View {
+        WPComEmailLoginView(viewModel: .init(siteURL: "https://test.com",
```

Comment:
nit: I think it's better to use a domain registered for this purpose, like `example.com`

Reply:
TIL about this! Updated in d64faf3137b7ced8ab582fe69f48e702f2c2473b.